waspc/data/Generator/templates/server/src/apis/types.ts

```diff
@@ -0,0 +1,20 @@
+{{={= =}=}}
+
+import prisma from '../dbClient.js'
+
+{=# isAuthEnabled =}
+import { {= userEntityUpper =} } from '../entities'
+{=/ isAuthEnabled =}
+
+{=# apiRoutes =}
+export type {= name =}Context = {
+  {=# isUsingAuth =}
+  user: {= userEntityUpper =},
+  {=/ isUsingAuth =}
+  entities: {
+    {=# entities =}
+    {= name =}: typeof prisma.{= prismaIdentifier =},
+    {=/ entities =}
+  }
+}
```

Comment:
We have a type for this in our internal types: https://github.com/wasp-lang/wasp/blob/cb691c8dd7348553bee123b2cdcf05d75847a0b5/waspc/data/Generator/templates/server/src/_types/index.ts#L35.

Reply:
Sure, let me know if I can help (but you should be able to just import and use it).